### PR TITLE
feat(cli): Show top 5 vulnerabilities by score in html report for namespace

### PR DESCRIPTION
**Pull request by xyoxo**
Resolves https://github.com/aquasecurity/starboard/issues/459

### CODE REVIEW

**Review by danielpacak — CHANGES_REQUESTED** [review]
👋 Good job @xyoxo I tested on my end and the report looks good. I left few comments and suggestions to look at.

File: pkg/report/templates/types.go [hunk 1]
```diff
@@ -26,6 +26,15 @@ type NamespaceReport struct {
 
 	Top5VulnerableImages []v1alpha1.VulnerabilityReport
 	Top5FailedChecks     []CheckWithCount
+	Top5Vulnerability    []Vulnerability
+}
+
+type Vulnerability struct {
+	ID                string
```
[the remark this answers is not in the document]
Comment (xyoxo) [reply pkg/report/templates/types.go:33]: I think that'll be better.

File: pkg/plugin/trivy/model.go [hunk 2]
```diff
@@ -20,6 +20,11 @@ type Vulnerability struct {
 	Layer            Layer             `json:"Layer"`
 	PrimaryURL       string            `json:"PrimaryURL"`
 	References       []string          `json:"References"`
+	Cvss             map[string]*CVSS  `json:"CVSS"`
+}
+
+type CVSS struct {
+	V3Score float64 `json:"V3Score,omitempty"`
```
[the remark this answers is not in the document]
Comment (xyoxo) [reply pkg/plugin/trivy/model.go:27]: I think, making it pointer will fix it. Then, if the json output is empty, we'll get nil value instead of 0.0.

**Review by danielpacak — APPROVED** [review]
LGTM!

**Pull request merged**